src/SourceGenerators/Uno.UI.SourceGenerators/DependencyObject/DependencyObjectGenerator.cs

```diff
@@ -225,9 +225,9 @@ private void GenerateIBinderImplementation(INamedTypeSymbol typeSymbol, Indented
 			private void WriteToStringOverride(INamedTypeSymbol typeSymbol, IndentedStringBuilder builder)
 			{
 				var hasNoToString = typeSymbol
-					.GetMethods()
+					.GetMethodsWithName("ToString")
 					.Where(m => IsNotDependencyObjectGeneratorSourceFile(m))
-					.None(m => m.Name == "ToString");
+					.None();
```

Comment:
I'm not sure if:

```csharp
				var hasNoToString = !typeSymbol
					.GetMethodsWithName("ToString")
					.Any(m => IsNotDependencyObjectGeneratorSourceFile(m));
```

can be faster than

```csharp
				var hasNoToString = typeSymbol
					.GetMethodsWithName("ToString")
					.Where(m => IsNotDependencyObjectGeneratorSourceFile(m))
					.None();
```

Do you know if there is a performance difference?

Reply:
It's likely faster by a small margin, but likely will allocate less :) Good catch!

Reply:
Will push to a separate PR